backend/services/useradm/api/http/router.go

```diff
@@ -0,0 +1,98 @@
+package http
+
+import (
+	"net/http"
+	"os"
+
+	"github.com/gin-gonic/gin"
```

Comment:
The linter wants some "air" here.
```suggestion
	"github.com/gin-gonic/gin"

```